bankdroid-legacy/src/main/java/com/liato/bankdroid/utils/ExceptionUtils.java

```diff
@@ -48,6 +39,27 @@
         return returnMe;
     }
 
+    @Nullable
+    private static <T extends Throwable> T createWrapperException(T wrap_me) {
```

Comment:
I can agree that underline methods and parameter names are more readable, especially in test cases. However, in the main code base, please follow the java naming convention.